src/shell/dotnet/Shell/App.xaml.cs

```diff
@@ -170,5 +172,16 @@ private void InjectMessageRouterConfig()
                     
                 """);
     }
+   
+    private void InjectFDC3()
+    {
+        string bundle = @"fdc3-iife-bundle.js";
+        string processPath = Environment.ProcessPath;
+        string folder = Path.GetDirectoryName(processPath);
+        string path = Path.Combine(folder, @"Preload\", bundle);
```

Comment:
This no longer works as the bundle is now an embedded resource. The only reason you're not getting errors is because the file is there as leftover from previous builds. Use https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getmanifestresourcestream?view=net-6.0#system-reflection-assembly-getmanifestresourcestream(system-string)

Reply:
Yes, I'm currently working on it. I pushed what I had yesterday in order to rebase my branch.

Reply:
Bundle is being accessed from the Embedded Resource now.